generated/nidcpower/nidcpower_library.h

```diff
@@ -118,14 +118,14 @@ class NiDCPowerLibrary : public nidcpower_grpc::NiDCPowerLibraryInterface {
   ViStatus GetExtCalLastDateAndTime(ViSession vi, ViInt32* year, ViInt32* month, ViInt32* day, ViInt32* hour, ViInt32* minute);
   ViStatus GetExtCalLastTemp(ViSession vi, ViReal64* temperature);
   ViStatus GetExtCalRecommendedInterval(ViSession vi, ViInt32* months);
-  ViStatus GetLCRCompensationData(ViSession vi, ViConstString channelName, ViInt32 compensationDataSize, ViInt8 compensationData[]);
+  ViStatus GetLCRCompensationData(ViSession vi, ViConstString channelName, ViInt32 compensationDataSize, ViAddr compensationData[]);
```

Comment:
The proto file is right to say the parameter is `bytes` but this is definitely wrong. It's either a `ViInt8[]` or just a simple scalar `ViAddr`. What do we do to fix this? @DavidCurtiss @phindman @reckenro

Reply:
Daniel Jaramillo, @alexdubois-ni, and @marcoskirsch discussed how to address the ViInt8[] vs ViAddr inconsistency, and the hapigen plugins were modified in [AzDO PR 377136](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/377136).

It was noted that some plugin-specific metadata changes might be necessary.  The type of metadata change suggested is to turn the `type` back to `i8` (or the equivalent for that plugin).  This sort of change was made to the CXP-specific metadata [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/379352?_a=files&path=/src/nidcpower/nidcpowerBaseDistribution/nidcpowerAPIMetadata/source/nidcpowerAPIMetadata/cxp/functions/functions.hapigen).

Reply:
Posted a [fix for the gRPC expander](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/381187).

Reply:
Also, Ryan's fix yesterday to the expander went in, and this uses it now. The signatures seem correct.